src/useObserver.ts

```diff
@@ -4,15 +4,31 @@ import { printDebugValue } from "./printDebugValue"
 import { isUsingStaticRendering } from "./staticRendering"
 import { useForceUpdate, useUnmount } from "./utils"
 
-export function useObserver<T>(fn: () => T, baseComponentName = "observed"): T {
+export type ForceUpdateHook = () => () => void
+
+export function useObserver<T>(
+    fn: () => T,
+    baseComponentName?: string,
```

Comment:
Please, the default name should stay here so it's more apparent. Do not hide down in the code.

Reply:
And actually same for that added option. It should be in here and down in the code. And you can rename that option to `useForceUpdate` directly, no need to call it a custom.

Reply:
done

Reply:
Thanks, much better.